src/components/pages/AppSubscription/AppSubscriptionDetailOverlay/index.tsx

```diff
@@ -234,6 +243,30 @@ const AppSubscriptionDetailOverlay = ({
     }
   }
 
+  const getDialogIntro = () => {
+    if (type === SubscriptionTypes.APP_SUBSCRIPTION) {
+      return t('content.appSubscription.detailOverlay.description')
+    } else {
+      return ti('serviceSubscription.detailOverlay.description')
+    }
+  }
+
+  const getSnackbarSeverity = () => {
+    if (tenantUrlResponse === TenantUrlState.SUCCESS) {
+      return SuccessErrorType.SUCCESS
+    } else {
+      return SuccessErrorType.ERROR
+    }
+  }
+
+  const getSnackbarDesc = () => {
+    if (tenantUrlResponse === TenantUrlState.SUCCESS) {
+      return t('content.appSubscription.detailOverlay.tenantUrlSuccessMsg')
+    } else {
+      return t('content.appSubscription.detailOverlay.tenantUrlErrorMsg')
+    }
+  }
```

Comment:
please write the three functions above simpler.
Instead of

`const getSomething = () => { if (condition) { return x } else { return y } }`

write

`const getSomething = () => condition ? x : y`

Reply:
@oyo I tried but due to code smells errors as we can't use ternary operator more than 15 times so use this way.